### PR TITLE
[CARBONDATA-2115] Docummentation - Scenarios in which aggregate query is not fetching …

### PR DESCRIPTION
**Pull request by sgururajshetty**
Added the FAQ on Scenarios in which aggregate query is not fetching

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

File: docs/faq.md [hunk 1]
```diff
@@ -141,4 +142,40 @@ INSERT INTO TABLE carbon_table SELECT id, city FROM source_table;
 
 When the column type in carbon table is different from the column specified in select statement. The insert operation will still success, but you may get NULL in result, because NULL will be substitute value when conversion type failed.
 
+## Why aggregate query is not fetching data from aggregate table?
+Following are the aggregate queries that won’t fetch data from aggregate table:
```
Comment (chenliang613) [inline docs/faq.md]: Please directly mention : in 1.3.0, not supporting the three cases.

**Pull request closed without merging**